packages/DApp/src/components/card/CardVote/CardVote.tsx

```diff
@@ -23,10 +26,19 @@ interface CardVoteProps {
 
 export const CardVote = ({ room, hideModalFunction }: CardVoteProps) => {
   const { account } = useEthers()
+  const [previousAccount, setPreviousAccount] = useState(account)
   const [showVoteModal, setShowVoteModal] = useState(false)
   const [showConfirmModal, setShowConfirmModal] = useState(false)
   const [proposingAmount, setProposingAmount] = useState(0)
   const [selectedVoted, setSelectedVoted] = useState(voteTypes['Add'].for)
+  const [sentVotesFor, setSentVotesFor] = useState(0)
+  const [sentVotesAgainst, setSentVotesAgainst] = useState(0)
+  const [voted, setVoted] = useState<null | boolean>(null)
+
+  if (previousAccount !== account) {
+    setPreviousAccount(account)
+    setVoted(null)
```

Comment:
nice catch

Reply:
I didn't find any reference to `previousAccount`, so unless we need it, we can just run an effect when account changes.

Reply:
> I didn't find any reference to `previousAccount`, so unless we need it, we can just run an effect when account changes.

No other ref then in the condition right?

Addressed at https://github.com/status-im/community-dapp/pull/31/commits/51a20180c8948a743f7bbe11f16145b4fcc701f9.